### PR TITLE
Add testcase for RHMI version metric

### PR DESCRIPTION
**Pull request by jjaferson**
# Description

Test case for RHMI version metric

 https://issues.redhat.com/browse/INTLY-7394

### CODE REVIEW

File: test-cases/tests/monitoring/m02-verify-rhmi-version-metric-is-exposed.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+tags:
+  - 2.3.0
+
+estimate: 15min
```
Comment (pawelpaszki) [inline test-cases/tests/monitoring/m02-verify-rhmi-version-metric-is-exposed.md:5]:
```suggestion
estimate: 15m
```

Comment (jjaferson) [reply test-cases/tests/monitoring/m02-verify-rhmi-version-metric-is-exposed.md]: done

**Pull request merged**